packages/api-headless-cms/src/plugins/graphql/installResolver/install.ts

```diff
@@ -0,0 +1,92 @@
+import mdbid from "mdbid";
+import { ErrorResponse, Response } from "@webiny/graphql";
+import { GraphQLFieldResolver } from "@webiny/graphql/types";
+
+const initialEnvironment = {
+    id: mdbid(),
+    name: "Production",
+    description: "Ready to go live",
+    createdFrom: null
+};
+
+const initialEnvironmentAlias = {
+    slug: "production",
+    name: "Production",
+    description: "Ready to go live"
+};
+
+const initialContentModelGroup = {
+    name: "E-commerce",
+    slug: "e-commerce",
+    description: "A generic group for e-commerce",
+    icon: "fas/star"
+};
+
+export const install: GraphQLFieldResolver = async (root, args, context) => {
+    const {
+        CmsSettings,
+        CmsEnvironment,
+        CmsEnvironmentAlias,
+        CmsContentModelGroup
+    } = context.models;
+
+    const cmsEnvironment = new CmsEnvironment();
+    const cmsEnvironmentAlias = new CmsEnvironmentAlias();
+    const cmsContentModelGroup = new CmsContentModelGroup();
+
+    try {
+        const settings = await CmsSettings.load();
+        if (await settings.data.installed) {
+            return new ErrorResponse({
+                code: "CMS_INSTALL_ABORTED",
+                message: "Cms is already installed."
+            });
+        }
+        // Create a production environment
+        const cmsEnvironmentProduction = await cmsEnvironment.populate({
+            id: initialEnvironment.id,
+            name: initialEnvironment.name,
+            description: initialEnvironment.description,
+            createdFrom: initialEnvironment.createdFrom
```

Comment:
You don't have this prop, right? Anyways, for the initial production environment, it should be `null`.